src/main/groovy/com/google/protobuf/gradle/internal/AndroidSourceSetFacade.groovy

```diff
@@ -0,0 +1,54 @@
+package com.google.protobuf.gradle.internal
+
+import groovy.transform.CompileStatic
+import com.android.build.api.dsl.AndroidSourceSet
+import com.android.build.gradle.api.AndroidSourceSet as DeprecatedAndroidSourceSet
+import org.gradle.api.plugins.ExtensionAware
+import org.gradle.api.plugins.ExtensionContainer
+
+@CompileStatic
+interface AndroidSourceSetFacade {
```

Comment:
Hmm or maybe use CompileDynamic. agp have the same api for all things what we are need.